### ISSUE TITLE
[Bug Report]: Stuck on the change grip interface

### ISSUE THREAD

**Issue by Link-HXX** [ndeadly/MissionControl#711]
### Switch Firmware Version

16.1.0 (Latest)

### Atmosphère Version

1.5.5 (Latest)

### Mission Control Version

0.9.3 (Latest)

### Boot Method

Fusée

### Issue Description

When matching and connecting a new handle in changing the grip, the pairing cannot be successful, and switch will be stuck on the changing grip interface, and the only option is to force shutdown. 
 I am connected to ns pro controller.  The switch stuck after long pressing the controller pairing button for a few seconds.

**Comment by ndeadly** [comment]
Yeah, this has been reported to me a few times already. I've been looking into it. See https://github.com/ndeadly/MissionControl/issues/696 for more info, including a workaround.

**Comment by Link-HXX** [comment]
Uh-huh.  Solved temporarily according to the method.Thanks for your reply, take it easy and have a great weekend

**Comment by Link-HXX** [comment]
Uh-huh.  Solved temporarily according to the method.Thanks

**Comment by ndeadly** [comment]
Should be fixed now, can you verify?

[MissionControl-0.9.3-develop-913c89b.zip](https://github.com/ndeadly/MissionControl/files/12451373/MissionControl-0.9.3-develop-913c89b.zip)

**Comment by Link-HXX** [comment]
It has been tested without any problems.  Thanks for your contribution.

**Comment by ndeadly** [comment]
Cool, I will push out the fix in a new release when I have a chance.
